spec/models/odk/response_saver_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
```

Comment:
This test simulates this:
![image](https://user-images.githubusercontent.com/989482/154147479-084f78b9-bc07-4a5e-ab62-fd65a45566ee.png)